backend/Application/DTOs/Admin/RegisterResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.DTOs.Admin
+{
+    public class RegisterResponse
```

Comment:
This will change in the future since what it should retrieve in the future is the token, the HasError I don't consider it needed since the status code will be there, The RegisterResponse should retrieve Register object, in this case, the Admin that has been just created, the token that will be a string, and the ResultMessage looks good for me here.